### ISSUE TITLE
should show user friendly error page in case of invalid / forbidden item

### ISSUE THREAD

**Issue by OriHoch** [Beit-Hatfutsot/dbs-front#333]
### reproduction steps
* open a forbidden or otherwise invalid item
  * for example (at time of writing): http://test.dbs.bh.org.il/image/mister-cohen-mrs-cohen-nee-orourke-master-cohen

##### expected
* error page, perhaps similar or the same as the 404 error page: http://test.dbs.bh.org.il/image/foo-bar-baz
* should provide some options to continue / resolve the problem (404 page should also have this..)

##### actual
* 
![image](https://cloud.githubusercontent.com/assets/1198854/24189270/f4545b90-0eec-11e7-8f18-2bf640dc553d.png)

**Comment by OriHoch** [comment]
@Netush  - perhaps some design is needed..
@TheGrandVizier - please prioritize

**Comment by TheGrandVizier** [comment]
@OriHoch  this shouldnt happen because the user shouldnt reach that far.
This item should not be returned as a search result, and should not be migrated to start with.

**Comment by OriHoch** [comment]
@TheGrandVizier  - you are correct, but there will always be some problem
now it's forbidden image, tomorrow it will be another problem..
we should have consistent and user friendly error pages

**Comment by TheGrandVizier** [comment]
@OriHoch agreed, but, does it matter that it's 403 or 404 to the user? I think the 404 page can be used in this case.

**Comment by OriHoch** [comment]
sure, if that's a good enough solution for you, that's fine

IMO there should be minimum of 2 error pages, and both should include follow-up to the user - 
* 404 page - in case of item or page that doesn't exist
  * should include follow-up (e.g. search box / suggested links)
* general server error page - in case of forbidden / invalid / other unexpected error
  * should show the error message (as returned from backend)
  * should tell the user the problem was reported to us and we will fix it
  * include follow-up (e.g. in case of search results, perhaps suggest to go back to other results)

But, of course, we need to prioritize..

**Comment by TheGrandVizier** [comment]
prioritized.

**Comment by Netush** [comment]
I agree that the these messages should be more visible and friendly.
Just a reminder, there was a design for the error messages but there was a decision to do something more easy - ready made.

**Comment by OriHoch** [comment]
@Netush - please add here these designs

**Comment by nuritgazit** [comment]
Ok, let's end the dicussion:
For now let's use the design of the 404 page for such cases (= there is no content in the page).
For the other cases where we need to tell something to the user, we can still use the existing error messages that appear on the top of the page. in the future we will change it to the designed error message
